### PR TITLE
Own Review

### PR DESCRIPTION
**Pull request by AmaliaGM**
I did a sushi product presentation. Included a nav bar at the page top with logo and a hamburguer button to the right, Introduced icons from a web page in order to connect menu links (hamburguer button) to this and create a responsive page, as I did with contact information also.
We added a product demonstration and prices chart.

### CODE REVIEW

**Review by AmaliaGM — COMMENTED** [review]
You did a great job.  Your commit messages could be more descriptive of your work. The icons could be aligned with the rest of the file (centered) 
Keep up the good work!

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @AmaliaGM, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

Your comment was professional and polite ✔️

## Aspects to improve ♻️

- [ ] You should use inline comments so the author of the project can easily understand the context of your comment. [Check this link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md)
- [ ] Remember to add a minimum of 2 comments as it is a requirement for this project. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AmaliaGM** [comment]
@leonardodiasb Thanks for your review.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        id="email"
+        type="email"
+        placeholder="Enter your email address"
+        required
+      />
+      <input id="submit" type="submit" value="Let's Eat" class="btn" />
+    </form>
+  </section>
+
+  <div class="container">
```
Comment (AmaliaGM) [inline index.html:54]: You could align all the elements to the titles it would look nice

File: README.md [hunk 2]
```diff
@@ -1 +1,73 @@
...
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Author1**
```
Comment (AmaliaGM) [inline README.md:43]: You could addapt your readme file to your proyect it woul make people know and appreciate your work better.

**Review by AmaliaGM — COMMENTED** [review]
Your page looks very nice and has some little details that would make it look better!
-Please take a look at the changes I´ve detailed for you.
-Have in mind also that commit messages are in order for someone else to know about your work at that period of time so it would be nice for you to give a more descriptive name to them.
Over all you´re doing a good job. Keep it up!
Happy codding =)

**Review by KayLemba — APPROVED** [review]
# Project Approved 🚀

Hi @AmaliaGM   🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights 
- You included an explanation, a "why", for each requested change 🎉 
-  You phrased the comments politely, as suggestions instead of orders 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- Kindly include emojis and gifs to make your code review more human-friendly 🙂 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**